### PR TITLE
feat: Adds Governance and Foundation pages

### PR DESCRIPTION
**Pull request by dgca**
### What changed?

- Adds Governance and Foundation pages

---

<sub>**Reminder:** If content (docs, blogs, pages) is moved or renamed, please ensure there are no broken links.</sub>

### CODE REVIEW

File: components/Governance/GovernanceFAQ/GovernanceFAQ.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    defaultMessage: "Governance FAQ",
+  },
+  mergeTitle: {
+    id: "governanceFaq.mergeTitle",
+    defaultMessage: "Who has merge permissions?",
+  },
+  mergeDescription: {
+    id: "governanceFaq.mergeDescription",
+    defaultMessage:
+      "Quam diu etiam furor iste tuus nos eludet? Me non paenitet nullum festiviorem excogitasse ad hoc. Etiam habebis sem dicantur magna mollis euismod.",
```
Comment (dgca) [inline components/Governance/GovernanceFAQ/GovernanceFAQ.tsx:16]: Need copy for FAQs.

File: components/Foundation/FoundationFAQ/FoundationFAQ.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    defaultMessage: "Foundation FAQ",
+  },
+  applicationTitle: {
+    id: "foundationFAQ.applicationTitle",
+    defaultMessage: "What are the requirements to apply for a grant?",
+  },
+  applicationDescription: {
+    id: "foundationFAQ.applicationDescription",
+    defaultMessage:
+      "Quam diu etiam furor iste tuus nos eludet? Me non paenitet nullum festiviorem excogitasse ad hoc. Etiam habebis sem dicantur magna mollis euismod.",
```
Comment (dgca) [inline components/Foundation/FoundationFAQ/FoundationFAQ.tsx:16]: Need copy for FAQs.

File: pages/community/foundation/index.tsx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+          }}
+          textAlign="center"
+          maxW="container.md"
+          mx="auto"
+        >
+          The Foundation guides Iron Fish towards premier crypto privacy.
+          Through partnerships, grants, and events, it boosts ecosystem growth,
+          emphasizing community-driven{" "}
+          <chakra.span textDecoration="underline">governance</chakra.span> and
+          fund allocation.
```
Comment (dgca) [inline pages/community/foundation/index.tsx:153]: i18n

File: shared/navLinks/links.ts [hunk 4]
```diff
@@ -131,6 +131,12 @@ export const links = [
         href: "/community/grants",
         image: "/images/nav/community-grants.svg",
       },
+      {
+        title: messages.foundationTitle,
+        description: messages.foundationDescription,
+        href: "/community/foundation",
+        image: "/images/nav/community-grants.svg",
```
Comment (dgca) [inline shared/navLinks/links.ts:138]: Need image.

File: shared/navLinks/messages.ts [hunk 5]
```diff
@@ -200,4 +200,12 @@ export const messages = defineMessages({
     id: "nav.bugBountyDescription",
     defaultMessage: "Participate in our cash incentivized bug bounty.",
   },
+  foundationTitle: {
+    id: "nav.foundationTitle",
+    defaultMessage: "Foundation",
+  },
+  foundationDescription: {
+    id: "nav.foundationDescription",
+    defaultMessage: "FILL THIS OUT FILL THIS OUT FILL THIS OUT",
```
Comment (dgca) [inline shared/navLinks/messages.ts:209]: Need content.

File: pages/community/governance/index.tsx [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+      "The Foundation has launched a Discourse website and put out a public call for governance submissions.",
+  },
+  submitProposal: {
+    id: "governance.submitProposal",
+    defaultMessage: "Submit Proposal",
+  },
+  communityDriven: {
+    id: "governance.communityDriven",
+    defaultMessage:
+      "Iron Fish has always been a <underline>community driven project.</underline> Since mainnet launch, our focus has been on how to continue bolstering Iron Fish&apos;s decentralization and resilience.",
```
Comment (dguenther) [inline pages/community/governance/index.tsx:41]: The escaped character is showing up literally for some reason: 

image

**Review by dguenther — APPROVED** [review]

**Pull request merged**